clojure-mode.el

```diff
@@ -191,38 +191,55 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
           (and (listp value)
                (cl-every 'stringp value))))
 
+(defcustom clojure-mode-keymap-prefix "C-c C-r"
```

Comment:
The refactoring prefix shouldn't really be customizable in this manner. A major mode's keybindings should always start with `C-c`, no exceptions. Only the `C-r` part might be made customizable, but I don't see much value in doing this.